assets/app/views/project.html

```diff
@@ -17,7 +17,24 @@
         <div class="row">
           <div class="col-md-12">
             <div class="tile">
-              <h2 class="service osc-object" ng-init="numPorts = service.spec.ports.length" kind="Service" resource="service">{{serviceId}}
+
+              <!-- Routes -->
+              <div ng-if="displayRouteByService[serviceId]" ng-init="route = displayRouteByService[serviceId]">
```

Comment:
@liggitt I have osc-object working, but it's using a generic object-describer that doesn't show important information like host. I can add routes to kubernetes-ui/object-describer. Do you want to do that under this PR or another?